readme.md

```diff
@@ -371,6 +371,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-check-orb](https://circleci.com/orbs/registry/orb/foo-software/lighthouse-check)** - A CircleCI Orb to run Lighthouse in a workflow, featuring Slack notifications and report upload to S3.
 * **[andreasonny83/lighthouse-ci](https://github.com/andreasonny83/lighthouse-ci)** - Run Lighthouse and assert scores satisfy your custom thresholds.
 * **[GoogleChrome/lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci)** - (**official**) Automate running Lighthouse for every commit, viewing the changes, and preventing regressions.
+* **[Lighthouse Data Studio Connector](https://datastudio.google.com/data?search=lighthouse)** - A Data Studio Connector that connects to Foo's public API to display Lighthouse charts as seen in [this example report](https://datastudio.google.com/u/0/reporting/1J8Yz9rLDAXQWDiSEtLrJDWxq79DMmIUB/page/HP1TB).
```

Comment:
Thanks for creating this. Pretty cool.

I do kinda think it's weird that there's a data studio connector named "Lighthouse" when it's using Foo's API and Foo's infrastructure. 

Since Lighthouse doesn't have a canonical service, I wouldn't really expect a "Lighthouse" connector ever...  (There's currently a PageSpeed Insights connector, and while it does have Lighthouse data in it, the connector isn't terribly useful. It looks like the data offered by your connector is definitely more useful to Data Studio workflows.)

Would you be okay with tweaking the name of your connector to something like "Lighthouse via Foo"

Reply:
Thanks for the feedback @paulirish. Yes, I was on the fence about the naming. I definitely get your point. Currently, when the Data Studio search displays `Lighthouse by Foo` and then the description of what it is below it `Lighthouse data from Foo's Lighthouse public API`. If you're using a Data Studio connector, I think you'd be more interested in what the thing (from the name) and not really the service that is providing the thing.

I'll think about renaming it. It would then display Lighthouse via Foo by Foo` 🤷

I think we can just close this for now. Sorry to waste your time.

Reply:
for reference:
![image](https://user-images.githubusercontent.com/39191/88234568-f9841900-cc2d-11ea-95eb-b6fa0e48d07a.png)

FWIW, I'm cool with having the link to the connector here in the readme.. 

> Yes, I was on the fence about naming. I definitely get your point. 

yeah sounds we're looking at it the same way. :)

Hypothetically the connector could be called "Webperf Monitoring" and retain the existing "By Foo" and existing description. I think that'd work well. ... er.. The search wouldn't pick it up. That's annoying

Okay in that case i'd go back to "Lighthouse (via Foo)"

![image](https://user-images.githubusercontent.com/39191/88235002-d312ad80-cc2e-11ea-8155-8c3c657232e7.png)

not amazing but seems ok given the constraints.

Reply:
Ya, that sounds good with parenthesis. Unfortunately the approval process is quite lengthy with quite a bit back fourth. Took a couple months to get this through. I respect the sentiment, but I wouldn't look at it as a performance tool only, since the we run all Lighthouse Audits. Anyways, thanks @paulirish. I'll create a new PR once I get the name changed perhaps.